client/app/create-recipe/page.tsx

```diff
@@ -38,6 +38,31 @@ const CreateRecipe: FC = () => {
     },
   });
 
+  const fetchProcessedBase64 = async (imageUrl: string) => {
+    try {
+      const response = await fetch(`/api/image-process?imageUrl=${imageUrl}`);
+      const data = await response.json();
+
+      return data.base64;
+    } catch (error) {
+      console.log(error);
+      return null;
```

Comment:
do u want to return null here? The way you've written this the error is swallowed and no one will ever know if there was an error unless they have the console open.

Reply:
I got it. I’ll fix the error handling so it will throw an error.